src/App/App.tsx

```diff
@@ -136,300 +137,315 @@ const App = (props: AppProps) => {
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={`${EDIT_ORGANIZATION_URL}/:id`}
+                  component={ConnectedCreateEditOrgView}
+                />
+                {/* single organization view */}
+                <ConnectedPrivateRoute
+                  redirectPath={APP_CALLBACK_URL}
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={`${SINGLE_ORGANIZATION_URL}/:id`}
+                  component={ConnectedSingleOrgView}
+                />
+                {/* single organization view */}
+                <ConnectedPrivateRoute
+                  redirectPath={APP_CALLBACK_URL}
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={`${SINGLE_ORGANIZATION_URL}/:id`}
+                  component={ConnectedSingleOrgView}
+                />
+                {/* Practitioner listing page */}
+                <ConnectedPrivateRoute
+                  redirectPath={APP_CALLBACK_URL}
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={PRACTITIONERS_LIST_URL}
+                  component={ConnectedPractitionersListView}
+                />
+                {/** practitioner create view */}
+                <ConnectedPrivateRoute
+                  redirectPath={APP_CALLBACK_URL}
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={CREATE_PRACTITIONER_URL}
+                  component={ConnectedCreateEditPractitionerView}
+                />
+                {/** Practitioner edit view */}
+                <ConnectedPrivateRoute
+                  redirectPath={APP_CALLBACK_URL}
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={`${EDIT_PRACTITIONER_URL}/:id`}
+                  component={ConnectedCreateEditPractitionerView}
+                />
+                {/** Assign practitioners to organization view */}
+                <ConnectedPrivateRoute
+                  redirectPath={APP_CALLBACK_URL}
+                  disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                  exact={true}
+                  path={`${ASSIGN_PRACTITIONERS_URL}/:id`}
+                  component={ConnectedAssignPractitioner}
+                />
+                {/* tslint:disable jsx-no-lambda */}
+                <Route
+                  exact={true}
+                  path={APP_LOGIN_URL}
+                  render={routeProps => (
+                    <OauthLogin
                       providers={providers}
-                      oAuthUserInfoGetter={oAuthUserInfoGetter}
+                      authorizationGrantType={AuthGrantType}
+                      OAuthLoginPromptMessage={LOGIN_PROMPT}
```

Comment:
What caused this change?

Reply:
non-intuitive breakpoint for the diff,

Reply:
The bigger diff is around nesting the header and body sections, giving that a min-height so that it push the footer section to the bottom of the page.